magento/values.yaml

```diff
@@ -0,0 +1,64 @@
+## Bitnami Magento image version
+## ref: https://hub.docker.com/r/bitnami/magento/tags/
+##
+imageTag: 2.0.5-r1
```

Comment:
For some reason the image 2.0.5-r1 was not built. I have triggered the build and built it in any case.

I am still getting the same error:

```
kubectl logs -f hardy-guppy-magento-3000609057-17wap
harpoon INFO  Initializing apache
harpoon INFO  apache successfully initialized
harpoon INFO  Initializing magento
magento INFO  Preparing webserver environment...
magento INFO  Preparing PHP environment...
Error executing 'postInstallation': File '/opt/bitnami/php/conf/php.ini' does not exists
```